test/e2e/gke_node_pools.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/onsi/ginkgo"
 )
 
-var _ = framework.KubeDescribe("GKE node pools [Feature:GKENodePool]", func() {
+var _ = framework.SIGDescribe("cloud-provider", "GKE node pools [Feature:GKENodePool]", func() {
```

Comment:
I missed the fact you're already touching this test.
I'm addressing it in: https://github.com/kubernetes/kubernetes/pull/99403